error_handler.py

```diff
@@ -0,0 +1,21 @@
+import discord
+import asyncio
+
+class Error_handler():
+    def __init__(self):
+        self.__error_messages = {}
```

Comment:
A question: why do we gather error messages for each user? Do we actually need to do that?

Reply:
The bot deletes a user's error-messages once the user enters a valid input, or when they complete their order. If we don't separate these by user, when multiple users are creating orders, one of the users entering a valid input will delete everyone's error messages regardless of whether they have been resolved